src/current/_includes/releases/v24.2/v24.2.0-beta.2.md

```diff
@@ -0,0 +1,50 @@
+## v24.2.0-beta.2
+
+Release Date: July 24, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-0-beta-2-security-updates">Security updates</h3>
+
+- URLs in the following SQL statements are now sanitized of any secrets before being written to [unredacted logs]({% link v24.2/configure-logs.md %}#redact-logs). [#127360][#127360]
+- [`ALTER BACKUP SCHEDULE`]({% link v24.2/alter-backup.md %})
+- [`ALTER BACKUP`]({% link v24.2/alter-backup.md %})
+- [`ALTER CHANGEFEED SET sink`]({% link v24.2/alter-changefeed.md %})
+- [`BACKUP`]({% link v24.2/backup.md %})
+- [`COPY`]({% link v24.2/copy-from.md %})
+- [`CREATE CHANGEFEED`]({% link v24.2/create-changefeed.md %})
+- [`CREATE EXTERNAL CONNECTION`]({% link v24.2/create-external-connection.md %})
+- [`CREATE SCHEDULE FOR BACKUP`]({% link v24.2/create-schedule-for-backup.md %})
+- [`CREATE SCHEDULE FOR CHANGEFEED`]({% link v24.2/create-schedule-for-changefeed.md %})
+- [`EXPORT`]({% link v24.2/export.md %})
+- [`IMPORT INTO`]({% link v24.2/import-into.md %})
```

Comment:
Just verifying that you intended to remove `IMPORT` in addition to adding `IMPORT INTO`?

Reply:
apparently there is no more IMPORT page in the v24.2 docs!  TIL